### DISCUSSION TITLE
Migration from Discourse to Github Discussions

### DISCUSSION THREAD

**Discussion by JonasHelming** [eclipse-theia/theia#13313] [General]
Hi,

the Theia Team decided to fully switch to [Github Discussions](https://github.com/eclipse-theia/theia/discussions) (this forum), the Discourse Forum is deprecated. Please search the [archived discussions](https://github.com/eclipse-theia/discourse-forum-archive/discussions) and in this forum before you ask questions in this forum.

Best regards,
Jonas

**Comment by rcarteraz** [comment]
> Hi,
> 
> the Theia Team decided to fully switch to [Github Discussions](https://github.com/eclipse-theia/theia/discussions) (this forum), the Discourse Forum is deprecated. Please search the [archived discussions](https://github.com/eclipse-theia/discourse-forum-archive/discussions) and in this forum before you ask questions in this forum.
> 
> Best regards, Jonas

Hi -- It looks like you were able to migrate your discourse forum to GitHub discussions, how did you manage that?

**Reply by JonasHelming** [reply]
We used a script to transfer the data via the API

**Reply by noamross** [reply]
Hi! We're thinking of doing that, too. Any chance that script is or could be posted somewhere?

**Reply by JonasHelming** [reply]
I just checked. Unfortunately, they evolved into something else and I cannot recover the state I used for the transition. However, it was relatively easy to do, just export => import.
Sorry!
